openedx-unsupported/configuration
# [BTR-35] Update ecommerce BROKER_URL default value

This PR updates Ansible variables for ecommerce and ecommerce worker services.

During testing Koa native installation, we've noticed that values in `BROKER_HOST` value in `/edx/etc/ecommerce.yml` and `/edx/etc/ecomworker.yml` files is invalid and still contains URL for RabbitMQ.

In `/edx/var/log/ecomworker/edx.log` I found log records like following:
```
Dec  3 13:47:13 ubuntu-focal [service_variant=ecomworker][celery.worker.consumer.consumer][env:no_env] ERROR [ubuntu-focal  28069] [consumer.py:428] - consumer: Cannot connect to amqp://celery:**@127.0.0.1:5672//: [Errno 111] Connection refused.
Trying again in 32.00 seconds... (16/100)
```

After changing `BROKER_HOST` to the correct value and restarting worker, `/edx/var/log/ecomworker/edx.log`:
```
Dec  3 14:06:45 ubuntu-focal [service_variant=ecomworker][celery.worker.consumer.connection][env:no_env] INFO [ubuntu-focal  30780] [connection.py:24] - Connected to redis://:**@localhost:6379//
```

**JIRA tickets**:
- [BTR-35](https://openedx.atlassian.net/browse/BTR-35)

**Dependencies**:
- https://github.com/edx/ecommerce/pull/3271
- https://github.com/edx/ecommerce-worker/pull/112

**Testing instructions**:

Follow official [Native installation guide](https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/1969455764/Koa+Native+Open+edX+platform+Ubuntu+20.04+64+bit+Installation), but in `config.yml` add the following:
```
ECOMMERCE_BROKER_USERNAME: ''
ECOMMERCE_BROKER_PORT: 6379
ECOMMERCE_BROKER_TRANSPORT: 'redis'

ECOMMERCE_WORKER_BROKER_USERNAME: ''
ECOMMERCE_WORKER_BROKER_PORT: 6379
ECOMMERCE_WORKER_BROKER_TRANSPORT: 'redis'
```

After installation is finished, you should see `Connected to redis://:**@localhost:6379/` line in `/edx/var/log/ecomworker/edx.log`.
